pytest_fixtures/component/mcp.py

```diff
@@ -57,31 +57,30 @@ def _setup_mcp_server(target_sat, settings_obj, is_downstream=False):
     )
     assert target_sat.execute('firewall-cmd --reload').status == 0
     target_sat.ensure_podman_installed()
+    ca_mountpoint = '/app/ca.pem'
     if is_downstream:
         target_sat.podman_login(
             settings_obj.registry_stage_username,
             settings_obj.registry_stage_password,
             settings_obj.registry_stage,
         )
         registry = settings_obj.registry_stage
-        ca_mountpoint = '/opt/app-root/src/ca.pem'
     else:
         registry = settings_obj.registry
-        ca_mountpoint = '/app/ca.pem'
     if not target_sat.network_type.has_ipv4:
         target_sat.execute('podman network create --ipv6 ipv6')
 
     authfile_arg = f"--authfile {PODMAN_AUTHFILE_PATH}" if is_downstream else ""
     network_arg = "--network ipv6" if not target_sat.network_type.has_ipv4 else ""
 
-    pull_cmd = f'podman pull {authfile_arg} {registry}/{settings_obj.image_path}'.strip()
+    pull_cmd = f'podman pull {authfile_arg} {registry}/{settings_obj.image_path}:{tag}'.strip()
     assert target_sat.execute(pull_cmd).status == 0
 
     run_cmd = (
         f'podman run {network_arg} {authfile_arg} '
         f'-v /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem:{ca_mountpoint}:ro,Z '
         f'--name {container_name} -d --pull=never -it -p {settings_obj.port}:8080 '
-        f'{image_name} --foreman-url https://{target_sat.hostname} --host 0.0.0.0'
+        f'{image_name}:{tag} --foreman-url https://{target_sat.hostname} --host 0.0.0.0'
```

Comment:
We should also include options to mirror what the docs suggest
```suggestion
        f'{image_name}:{tag}:ro,Z --foreman-url https://{target_sat.hostname} --host 0.0.0.0'
```